File: Backend/src/main/java/com/feedle/feedleapi/Networking/DeletePostRequest.java
package com.feedle.feedleapi.Networking;

public class DeletePostRequest extends Request{
    private int postId;
    public DeletePostRequest(int postId)
    {
        super(RequestType.DeletePost);
        this.postId = postId;
    }

    public int getPostId() {
        return postId;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/DeletePostRequest.cs
using System.Text.Json.Serialization;

namespace FeedleDataTier.Network
{
    public class DeletePostRequest : Request
    {

        [JsonPropertyName("postId")] 
        public int PostId { get; set; }

        public DeletePostRequest(int postId) : base(RequestType.DeletePost)
        {
            this.PostId = postId;
        }
    }
}